binding/client.go

```diff
@@ -69,34 +69,28 @@ func (s Path) Inject(p Params) (out string) {
 }
 
 // NewClient Constructs a new client
-func NewClient(url string, token api.Login) (client *Client) {
+func NewClient(baseURL string) (client *Client) {
 	client = &Client{
-		baseURL: url,
-		token:   token,
+		BaseURL: baseURL,
 	}
 	client.Retry = RetryLimit
 	return
 }
 
 // Client provides a REST client.
 type Client struct {
-	// baseURL for the nub.
-	baseURL string
-	// addon API token
-	token api.Login
 	// transport
 	transport http.RoundTripper
+	// baseURL for the nub.
+	BaseURL string
+	// login API resource.
+	Login api.Login
```

Comment:
**hint**: renamed to be more accurate.  This is not a token but instead a Login REST resource.
Also, no value making the not-exported.  Simpler.